app/views/steps/case/has_codefendants/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<% title t('.page_title') %>
+<% step_header(path: crime_applications_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary(@form_object) %>
+
+    <%= step_form @form_object do |f| %>
+        <%= f.govuk_collection_radio_buttons :has_codefendants, @form_object.choices,
+                                             :value, legend: { tag: 'h1', size: 'xl' } %>
+
+        <%= f.continue_button(secondary: false) %>
```

Comment:
We want the save and come back later button in this step so let's remove `secondary: false`

I think you based some of your code on the `has_partner` step which admittedly has some unique properties given is the very first step in the journey, but that's ok, I think this and the step_header are the only peculiar things.